metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1662,11 +1664,12 @@ def _convert_to_application(self, result: Mapping[str, Any]) -> Application:
             application_url=_safe_get(result, 'application_url'),
             description=_safe_get(result, 'description'),
             name=_safe_get(result, 'name'),
-            id=_safe_get(result, 'id', default=''))
+            id=_safe_get(result, 'id', default='')
+        )
 
     def _convert_to_description(self, result: Mapping[str, Any]) -> ProgrammaticDescription:
         return ProgrammaticDescription(text=_safe_get(result, 'description'),
-                                       source=_safe_get(result, 'source'))
+                                       source=_safe_get(result, 'description_source'))
```

Comment:
From: https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/models/table_metadata.py#L86

Reply:
I think `description_source` has been defined in databuilder for programmatic description. I would prefer to change it here to match the databuilder implementation.

Reply:
I think `source` is more concise so I'd prefer that in a model with description already in the name, but at the very least common and databuilder should be in agreement about the field names!

Reply:
could we have a follow up todo to change the common repo to make it consistent?